live/app/components/identification-form.js

```diff
@@ -0,0 +1,73 @@
+import Ember from 'ember';
+import _ from 'lodash/lodash';
+
+// XXX from http://stackoverflow.com/a/46181/2120773
+function validateEmail(email) {
+
+  const regExp = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
OMG. 😱 Quelle horreur. May I... https://davidcel.is/posts/stop-validating-email-addresses-with-regex/ ?